readme.md

```diff
@@ -34,7 +34,7 @@ Kick off a run by passing `lighthouse` the URL to audit:
 lighthouse https://airhorner.com/
 ```
 
-Lighthouse will prettyprint a report to CLI. You can control the output format by passing flags.
+Lighthouse will write an HTML report to file. You can control the output format by passing flags.
```

Comment:
I think the point of the original statement is to point out what you get by default and call attention to the fact that there are flags to change that.

Agree it is a bit awkward. "Lighthouse will write an HTML report to file by default." to call attention to the fact that it's talking about the default behavior?

Also we could delete the line :)